.github/workflows/format.yml

```
@@ -1,29 +1,25 @@
 name: "format code"
 
-# run on pull requests to develop
-on:
-  pull_request:
-    branches:
-      - master
+on: push
 
 jobs:
   format:
     runs-on: ubuntu-latest
     steps:
       # check out HEAD on the branch
-      - name: checkout code
-        uses: actions/checkout@2
+      - uses: actions/checkout@v2
         with:
           ref: ${{ github.head_ref }}
+
       # format the latest commit
-      - name: format code
-        uses: purduesigbots/clang-format-action@1
+      - uses: purduesigbots/clang-format-action@1.0.2
```

```suggestion
      - uses: purduesigbots/clang-format-action@main
```
there's still bugs in the released version, use HEAD on the main branch to get the docker container building